.drone.yml

```diff
@@ -0,0 +1,37 @@
+matrix:
+  NAME:
+    - ukhomeofficedigital/cdp-ci
+  DOCKER_REPO:
+    - quay.io
```

Comment:
Looks like we aren't using any of these variables so I think all of the above should go.

Reply:
Yes. It's actually already been removed in CDMP-2366
[.drone.yml in CDMP-2366](https://github.com/UKHomeOffice/aws-budget-alerting/blob/CDMP-2366/.drone.yml)

Reply:
So to clarify my comment, I suggest we leave this as it is in this PR as the following one (CDMP-2366) will take care of fixing it. The following PRs are also queued behind the latter one: CDMP-2365, CDMP-2364, CDMP-2367